starter_code/views/album.ejs

```diff
@@ -0,0 +1,12 @@
+<a href="/"><button>BackHome</button></a>
```

Comment:
Maybe you can put this button in the layout and call it Home (instead of Back Home).